packages/shared/src/hooks/usePostContent.ts

```diff
@@ -35,13 +40,29 @@ const usePostContent = ({
   const { user } = useAuthContext();
   const { trackEvent } = useAnalyticsContext();
   const onPostClick = useOnPostClick({ origin });
+  const { commentsPermalink } = post;
+  const cid = ReferralCampaignKey.SharePost;
+  const { getShortUrl } = useGetShortUrl();
+  const { sharePost, closeSharePost } = useSharePost(origin);
+  const [, copyLink] = useCopyPostLink();
+  const trackShareEvent = (provider: ShareProvider) =>
+    trackEvent(
+      postAnalyticsEvent('share post', post, {
+        extra: { provider, origin: Origin.ShareBar },
```

Comment:
Is the origin ok here?
Not sure where we use this hook from, but might need to pass origin from top level somewhere?

Reply:
This might need adjustment, verifying now

Reply:
It was passed, I had forgotten to use the right one here